workflow_scripts/update_onnx_hub_manifest.py

```diff
@@ -0,0 +1,122 @@
+import glob
+import hashlib
+import json
+import os
+import re
+import bs4
+import markdown
+import pandas as pd
+import typepy
+from os.path import join, split
+
+
+# Acknowledgments to pytablereader codebase for this function
+def parse_html(table):
+    headers = []
+    data_matrix = []
+    rows = table.find_all("tr")
+    re_table_val = re.compile("td|th")
+    for row in rows:
+        td_list = row.find_all("td")
+        if typepy.is_empty_sequence(td_list):
+            if typepy.is_not_empty_sequence(headers):
+                continue
+            th_list = row.find_all("th")
+            if typepy.is_empty_sequence(th_list):
+                continue
+            headers = [row.text.strip() for row in th_list]
+            continue
+        data_matrix.append(list(row.find_all(re_table_val)))
+
+    if typepy.is_empty_sequence(data_matrix):
+        raise ValueError("data matrix is empty")
+
+    return pd.DataFrame(data_matrix, columns=headers)
+
+
+markdown_files = list(glob.glob(join("..", "**", "README.md"), recursive=True))[1:]
+manifest_object = []
+
+all_tables = []
+for markdown_file in markdown_files:

```

Comment:
We don't have any specification about how the ONNX models will be organized.
But there is a table in the https://github.com/onnx/models/blob/master/README.md which can be a starter to list all models.

Reply:
I pulled out all info from inner tables to generate this manifest these are the only issues encountered:

Unrecognized table columns in file ../vision/body_analysis/age_gender/README.md: ['Model (Caffe)' 'Download' 'ONNX version' 'Opset version' 'Dataset']
Unrecognized table columns in file ../vision/object_detection_segmentation/duc/README.md: ['Filename' 'Size' 'Details']
no model_with_data in file ../vision/body_analysis/ultraface/README.md
no model_with_data in file ../vision/body_analysis/ultraface/README.md
no model_with_data in file ../vision/object_detection_segmentation/yolov2-coco/README.md
malformed opset ONNX 9, ONNX.ML 1 in ../text/machine_comprehension/bidirectional_attention_flow/README.md
no model_with_data in file ../text/machine_comprehension/t5/README.md
no model_with_data in file ../text/machine_comprehension/t5/README.md

Reply:
I just checked some links in table in the README are all alive, as well as the onnx model link.
Do you know what causes the issues you mentioned?

Reply:
If you decide to make dependence on the table in the top-level README, we probably can add a CI pipeline to valid any change made on this README will not break this generator script.